cloud_testing/kotlin_poc/src/main/kotlin/ftl/config/YamlConfig.kt

```diff
@@ -191,7 +193,8 @@ class YamlConfig(
                 testMethods: $testMethods,
                 limitBreak: $limitBreak,
                 devices: $devices,
-                environmentVariables: $environmentVariables
+                environmentVariables: $environmentVariables,
+                directoriesToPull: $directoriesToPull
```

Comment:
directories to pull is Android only
https://cloud.google.com/sdk/gcloud/reference/alpha/firebase/test/ios/run

Reply:
Good to know, thanks